torch/testing/_internal/common_methods_invocations.py

```diff
@@ -18055,6 +18055,7 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
 reference_filtered_ops = [op for op in reduction_ops if op.ref is not None]
 reference_masked_ops = [op for op in reference_filtered_ops if op.name.startswith('masked.')]
 sparse_masked_reduction_ops = [op for op in sparse_reduction_ops if op.name.startswith('masked.')]
+signal_funcs = [op for op in op_db if 'signal' in op.name]
```

Comment:
I think this may not be necessary any more.

Reply:
Removed this and I'm searching the window ops using `op_db` in the tests.